src/kbmod/search/stack_search.cpp

```diff
@@ -288,7 +306,10 @@ static void stack_search_bindings(py::module& m) {
             .def("prepare_psi_phi", &ks::prepare_psi_phi, pydocs::DOC_StackSearch_prepare_psi_phi)
             .def("clear_psi_phi", &ks::clear_psi_phi, pydocs::DOC_StackSearch_clear_psi_phi)
             .def("get_results", &ks::get_results, pydocs::DOC_StackSearch_get_results)
-            .def("set_results", &ks::set_results, pydocs::DOC_StackSearch_set_results);
+            .def("set_results", &ks::set_results, pydocs::DOC_StackSearch_set_results)
+            .def("search_single_batch", &ks::search_single_batch, pydocs::DOC_StackSearch_search_single_batch)
+            .def("prepare_search", &ks::prepare_search, pydocs::DOC_StackSearch_prepare_batch_search)
+            .def("finish_search", &ks::finish_search, pydocs::DOC_StackSearch_finish_search);
```

Comment:
I think it could be worthwhile to include the (renamed) `extract_max_results` in the python interface for use in testing.

Reply:
I exposed it but stopped short of writing a test for it because there seems that `RESULTS_PER_PIXEL` from `common.h` is not being exposed anywhere. I think it would be valuable to have some python binding that shows these settings

I see that the test I wrote for the batch search assumes 8 results per pixel but it would be nicer if it could extract it straight from a `Config` object that binds those settings from the internal cpp headers. I can do a follow-up PR if you think that makes sense.